cobalt/browser/cobalt_content_browser_client.cc

```diff
@@ -47,6 +65,9 @@ std::string GetCobaltUserAgent() {
 blink::UserAgentMetadata GetCobaltUserAgentMetadata() {
   blink::UserAgentMetadata metadata;
 
+#define COBALT_BRAND_NAME "Cobalt"
+#define COBALT_MAJOR_VERSION "26"
+#define COBALT_VERSION "26.lts.0-qa"
   metadata.brand_version_list.emplace_back(COBALT_BRAND_NAME,
```

Comment:
Where do these macros (i.e. COBALT_BRAND_NAME) currently come from? It seems like we're just overriding existing values?

Reply:
Ah okay, didn't notice that they were already in this file before your change.